llvm/lib/Target/PowerPC/PPCMIPeephole.cpp

```diff
@@ -177,11 +177,6 @@ struct PPCMIPeephole : public MachineFunctionPass {
     if (skipFunction(MF.getFunction()))
       return false;
     bool Changed = simplifyCode();
-#ifndef NDEBUG
```

Comment:
There is a comment at line 19 saying that the verifier will be run. It should be removed if the verifier will not be run.

Reply:
Could we put this under EXPENSIVE_CHECKS rather than removing it?

Reply:
Totally removing this sounds not good to save compile time, functionality issue will be hidden. We have run machine verifier many times, why the one here matters?

Reply:
this `MF.verify`  cause Compile time regression with 502.gcc at BASE and PEAK from 62 seconds to 255 seconds

Reply:
Isn't the verifier run after every pass anyway if EXPENSIVE_CHECKS are enabled?

Reply:
I do not think so ,  the verifier only run after the passes  which has 

```
#if defined(EXPENSIVE_CHECKS)
  MF.verify();
#endif
```

Reply:
> Isn't the verifier run after every pass anyway if EXPENSIVE_CHECKS are enabled?

Tried and can confirm that this is the case.